src/pages/ProfileEditPage/ProfileEditPage.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react"
+
+import {
+  Flex,
+  StackDivider,
+  VStack,
+  useDisclosure,
+  useMediaQuery,
+} from "@chakra-ui/react"
+
+import ChangePWModal from "./components/Modal/ChangePWModal"
+import ProfileCard from "./components/Profile/ProfileCard"
+import ProfileIntroduction from "./components/Profile/ProfileIntroduction"
+// import ProfileTechStack from "./components/ProfileTechStack"
+import StyledButton from "./styles/StyledButton"
+
+const ProfileEditPage = () => {
+  const [isLargerThan500] = useMediaQuery("(min-width: 500px)")
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  const [profileInfo, setProfileInfo] = useState({
+    profileImageUrl: "",
+    nickname: "개발자",
+    career: "0년차",
+    introduction: "",
+    githubUrl: "",
+    blogUrl: "",
+  })
+
+  const handleSubmit = () => {
+    // TODO: 프로필 정보를 저장하는 api 요청
+    console.log(profileInfo)
+  }
+  return (
+    <VStack
+      w="80%"
+      h="100vh"
+      align="start"
+      m="auto">
```

Comment:
```suggestion
    <VStack
      w="80%"
      align="start"
      m="auto">
```
[P2] : 헤더 높이 반영 늦어져서 죄송해요 ㅠㅠ
이쪽에 높이는 안주셔도 될 것 같아요!